docs/releasenotes/index.rst

```diff
@@ -10,6 +10,14 @@ You can learn more about where we might be going with future releases by:
 * Stopping by the ``#clusterhq`` channel on ``irc.freenode.net``.
 * Visiting our GitHub repository at https://github.com/ClusterHQ/flocker.
 
+v1.3
+====
+
+* Flocker API has new endpoints to create, list, renew, and delete lease on a dataset.
```

Comment:
I'm not sure I understand this sentence, should there be another comma or an extra word between ``delete`` and ``lease``?